### PR TITLE
update instructor guide

### PR DESCRIPTION
**Pull request by wikfeldt**
with
- why we teach the lesson
- intended learning outcomes
- how to teach the lesson

### CODE REVIEW

File: guide.md [hunk 1]
```diff
@@ -5,7 +5,57 @@ permalink: /guide/
...
+- know how to share notebooks via mybinder
+- be able to relate to the possible pitfalls of using notebooks
+- be able to distinguish good and bad examples of notebook usage 
+
+## How to teach this lesson
+
+### Requirements
 
 These are the packages used in the lesson material and exercises:
 - jupyter
```
Comment (annefou) [inline guide.md:61]: Shouldn't we have jupyterlab too?

**Review by annefou — APPROVED** [review]
That is great! I just added a comment about the packages and whether we should add jupyterlab in the list.
Thanks!

File: guide.md (hunk 1 above)
Comment (wikfeldt) [reply guide.md:61]: good catch!

Comment (wikfeldt) [reply guide.md:61]: but jupyterlab is there

**Comment by wikfeldt** [comment]
merging then!

**Pull request merged**